plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/GradleEnclavePlugin.kt

```diff
@@ -493,16 +502,17 @@ class GradleEnclavePlugin @Inject constructor(private val layout: ProjectLayout)
             }
 
             target.afterEvaluate {
-                val runtimeType = conclaveExtension.runtime.get()
-
-                when (runtimeType) {
-                    RuntimeType.Gramine -> {
-                        target.artifacts.add(typeLowerCase, signedEnclaveGramineJarTask.get().archiveFile)
-                    }
-
-                    RuntimeType.GraalVM -> {
-                        target.artifacts.add(typeLowerCase, signedEnclaveJarTask.get().archiveFile)
+                try {
+                    when (RuntimeType.valueOf(conclaveExtension.runtime.get())) {
+                        RuntimeType.Gramine -> {
+                            target.artifacts.add(typeLowerCase, signedEnclaveGramineJarTask.get().archiveFile)
+                        }
+                        RuntimeType.GraalVM -> {
+                            target.artifacts.add(typeLowerCase, signedEnclaveJarTask.get().archiveFile)
+                        }
                     }
+                } catch (e: Exception) {
+                    /** Do nothing. Error condition is handled in [apply]. */
```

Comment:
What happens if you don't have this catch?

Reply:
The exception is printed an additional three times, once for each simulation, debug and release.